xrpl/models/requests/amm_info.py

```diff
@@ -0,0 +1,53 @@
+"""This request retrieves information about an AMM instance."""
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Dict, Optional
+
+from xrpl.models.amounts import Amount
+from xrpl.models.requests.request import Request, RequestMethod
+from xrpl.models.utils import require_kwargs_on_init
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class AMMInfo(Request):
+    """
+    This request retrieves information about an AMM instance.
+
+    Must provide either AMMHash or both Asset1 and Asset2 params.
+    """
+
+    AMMHash: Optional[str] = None
+    """
+    AMMHash is a hash that uniquely identifies the AMM instance.
+    """
+
+    Asset1: Optional[Amount] = None
```

Comment:
Why is an amount specified if details are desired about it?

Reply:
Asset1/Asset2 can accept a string that represents XRP in drops.